src/pages/PersonalisationSummaryPage.js

```diff
@@ -10,23 +10,15 @@ import icons from "../icons";
 class PersonalisationSummaryPage extends BasePersonalisationPage {
 
     previousStep(): void {
-        if (this.state.subpage == 0) {
-            this.props.history.goBack();
-        } else {
-            this.setState({subpage: 0});
-        }
+        this.props.history.goBack();
     }
 
     nextStep(): void {
-        this.setState({subpage: 0});
+        this.props.history.goBack();
```

Comment:
`NavigationMixin` was replaced with `History` mixin in the new react-router, which is what I've used here.